SETUP.md

```diff
@@ -1,4 +1,42 @@
-My App Setup Instructions
+Lansweeper Setup Instructions
 ===
 
-TODO: Write setup instructions
+To install the Lansweeper App, you must first create an app with a "Client ID" and "Client Secret".
+
+Once you've logged in, navigate to the __"Settings"__ (click on your avatar at the bottom left of the screen)
+
+[![](/docs/setup/setup-lansweeper-01.png)](/docs/setup/setup-lansweeper-01.png)
+
+After that, navigate to the __"Developer tools"__ the sidebar menu and choose __"All applications"__.
+
+[![](/docs/setup/setup-lansweeper-02.png)](/docs/setup/setup-lansweeper-02.png)
+
+Select the __"Add new application +"__ item in the sidebar or click the __"Add new application"__ button to proceed with creating the application.
+
+[![](/docs/setup/setup-lansweeper-03.png)](/docs/setup/setup-lansweeper-03.png)
+
+In the opened window, select __"Cloud Application"__.
+
+[![](/docs/setup/setup-lansweeper-04.png)](/docs/setup/setup-lansweeper-04.png)
+
+Then, click on __"Continue"__.
+
+Here, we fill out the following fields:
+1. Provide the __"Application name"__ as "Deskpro App" (or any preferred name);
+2. Copy the __"Сallback URL"__ on the "Settings" tab and paste it into the __"Allowed callback URL"__
+3. Choose __"Other"__ in the Integration dropdown
+4. In the __"Custom Integration Name"__ field, you can enter "Deskpro App" the same as in the "Application name" field.
+5. The __"Developed by"__ and __"Description"__ fields can be left blank. However, feel free to fill them in as you see fit.
+
+[![](/docs/setup/setup-lansweeper-05.png)](/docs/setup/setup-lansweeper-05.png)
+
+After filling in the fields, click __"Add Application"__. Then save the generated credentials and store them in a secure place.
+
+Ok, head back to Deskpro and enter your __Client ID__ and __Client secret__ into the app settings form.
```

Comment:
... which you got from the generated credentials in Lansweeper.

Reply:
added